src/Client/pages/Home.fs

```diff
@@ -7,6 +7,13 @@ open Fulma
 open Client
 open Client.AppPages
 
+let linkBlock text colorClass = 
+    let firstSymbol: char = Seq.head text
```

Comment:
Maybe just `let firstSymbol = text.[0]`? [Ultimate recommendation](https://stackoverflow.com/a/3878828/3232646) :D

Reply:
using head seems more idiomatic to me (for f#) 😄

Reply:
Alright